core/Cargo.toml

```diff
@@ -35,6 +35,7 @@ anyhow.workspace = true
 assert_matches = { version = "1", optional = true }
 backoff = { version = "0.4.0", features = ["tokio"] }
 base64.workspace = true
+bytes = "1"
```

Comment:
nit: I'd prefer to see the dep hoisted up into the top-level `Cargo.toml` and replaced with `bytes.workspace = true` here and in `aggregator/Cargo.toml`.